README.md

```diff
@@ -20,26 +20,83 @@ The Go library documentation can be found [here][libdocs].
 
 This library supports the following Go implementations:
 
-* Go 1.15
-* Go 1.16
-* Go 1.17
-* Go 1.18
-* Go 1.19
+- Go 1.15
+- Go 1.16
+- Go 1.17
+- Go 1.18
+- Go 1.19
 
 ## Installation
 
-To use twilio-go in your project initialize go modules then run:
+The recommended way to install `twilio-go` is by using [Go modules](https://go.dev/ref/mod#go-get).
 
-```bash
+If you already have an initialized project, you can run the command below from your terminal in the project directory to install the library:
+
+```shell
 go get github.com/twilio/twilio-go
 ```
 
-## Getting Started
+If you are starting from scratch in a new directory, you will first need to create a go.mod file for tracking dependencies such as twilio-go. This is similar to using package.json in a Node.js project or requirements.txt in a Python project. [You can read more about mod files in the Go documentation](https://golang.org/doc/modules/managing-dependencies). To create the file, run the following command in your terminal:
+
+```shell
+go mod init twilio-example
+```
+
+Once the module is initialized, you may run the installation command from above, which will update your go.mod file to include twilio-go.
+
+### Test your installation
+
+Try sending yourself an SMS message by pasting the following code example into a sendsms.go file in the same directory where you installed twilio-go. Be sure to update the accountSid, authToken, and from phone number with values from your Twilio account. The to phone number can be your own mobile phone number.
+
+```go
+package main
+
+import (
+  "encoding/json"
```

Comment:
This is a few spaces and not a tab character (the line below it is a tab) and might look funny in some text editors.